torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12240,7 +12270,12 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            supports_autograd=False,
            sample_inputs_func=sample_inputs_ormqr,
            error_inputs_func=error_inputs_ormqr,
-           decorators=[skipCUDAIfNoCusolver, skipCPUIfNoLapack],
+           decorators=[
+               skipCUDAIfNoCusolver,
+               skipCPUIfNoLapack,
+               # Resulting values are very large therefore atol is increased and rely on rtol
+               DecorateInfo(toleranceOverride({torch.float32: tol(atol=5000, rtol=1e-05)}),
+                            'TestCommon', 'test_compare_cpu')],
```

Comment:
This doesn't look right. If you increase the atol so much it's the same as testing nothing.

Reply:
Yes, This was to turn the atol 'off', the elements in question are roughly `1**10` so I was relying on the rtol.

* cpu   `3.71387269e+10`
* cuda `3.71387310e+10`

how should I proceed?

Reply:
then you should do `atol=0`, right?

Reply:
I thought atol and rtol were mutually exclusive. fixed it now